File: backend/src/main/java/com/ll/TeamProject/domain/user/entity/ForbiddenNickname.java
package com.ll.TeamProject.domain.user.entity;

import com.ll.TeamProject.global.jpa.entity.BaseTime;
import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Entity
@Getter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class ForbiddenNickname extends BaseTime {
    // BaseTime : id (BaseEntity, no setter), 생성/수정일

    @Column(unique = true)
    private String forbiddenName;
}

File: backend/src/main/kotlin/com/ll/TeamProject/domain/user/entity/ForbiddenNickname.kt
package com.ll.TeamProject.domain.user.entity

import com.ll.TeamProject.global.jpa.entity.BaseTime
import jakarta.persistence.Column
import jakarta.persistence.Entity

@Entity
class ForbiddenNickname(
    @Column(unique = true)
    var forbiddenName: String
) : BaseTime() {

    protected constructor() : this("")
}
